lib/ansible/modules/cloud/misc/helm_cli.py

```diff
@@ -0,0 +1,532 @@
+    description:
+      - Run `helm repo update` before the operation. Can be run as part of the package installation or as a separate step.
+    default: "false"
+    type: bool
+    version_added: "2.10"
+
+#Helm options
+  disable_hook:
+    description:
+      - Helm option to disable hook on install/upgrade/delete
+    default: False
+    type: bool
+    version_added: "2.10"
+  force:
+    description:
+      - Helm option to force reinstall, ignore on new install
+    default: False
+    type: bool
+    version_added: "2.10"
+  purge:
+    description:
+      - Remove the release from the store and make its name free for later use
+    default: True
+    type: bool
+    version_added: "2.10"
+  wait:
+    description:
+      - Wait until all Pods, PVCs, Services, and minimum number of Pods of a Deployment are in a ready state before marking the release as successful
+    default: False
+    type: bool
+    version_added: "2.10"
+  wait_timeout:
+    description:
+      - Timeout when wait option is enabled in second
+    default: 300
+    type: int
+    version_added: "2.10"
+'''
+
+EXAMPLES = '''
+# Deploy grafana with params version
+- helm_cli:
+    name: test
+    chart_ref: stable/grafana
+    chart_version: 3.3.8
+    tiller_namespace: helm
+    values:
+      replicas: 2
+
+# Remove test release
+- helm_cli:
+    name: test
+    state: absent
+    tiller_namespace: helm
+'''
+
+RETURN = """
+status:
+  type: complex
+  description: A dictionary of status output
+  returned: on success Creation/Upgrade/Already deploy
+  contains:
+    AppVersion:
+      type: str
+      returned: always
+      description: Version of app deployed
+    Chart:
+      type: str
+      returned: always
+      description: Chart name and chart version
+    Name:
+      type: str
+      returned: always
+      description: Name of the release
+    Namespace:
+      type: str
+      returned: always
+      description: Namespace where the release is deployed
+    Revision:
+      type: str
+      returned: always
+      description: Number of time where the release has been updated
+    Status:
+      type: str
+      returned: always
+      description: Status of release (can be DEPLOYED, FAILED, ...)
+    Updated:
+      type: str
+      returned: always
+      description: The Date of last update
+    Values:
+      type: str
+      returned: always
+      description: Dict of Values used to deploy
+stdout:
+  type: str
+  description: Full `helm` command stdout, in case you want to display it or examine the event log
+  returned: always
+  sample: ''
+stderr:
+  type: str
+  description: Full `helm` command stderr, in case you want to display it or examine the event log
+  returned: always
+  sample: ''
+command:
+  type: str
+  description: Full `helm` command built by this module, in case you want to re-run the command outside the module or debug a problem.
+  returned: always
+  sample: helm upgrade ...
+"""
+
+try:
+    import yaml
```

Comment:
Is that guaranteed to be available everywhere or should it be a requirement listed with the module?

Reply:
import yaml is guaranteed by the pyaml module, one of ansible depency

Reply:
Yes, but this module doesn't run on machines where ansible is installed...

Reply:
Interesting !
Can you give me more information on your setup ?
I test with ansible 2.7 and 2.8 on clean virtualenv and i can't reproduce your issue
```
$ pip list
Package      Version
------------ -------
ansible      2.8.5
asn1crypto   1.1.0
cffi         1.12.3
cryptography 2.7
Jinja2       2.10.3
MarkupSafe   1.1.1
pip          19.2.3
pycparser    2.19
PyYAML       5.1.2
setuptools   41.2.0
six          1.12.0
wheel        0.33.6
```

Reply:
This is a module, it gets run on remote hosts, not your Ansible host. If you install Ansible on machines you manage, you're doing something wrong.

Maybe you'll see what I mean once you add the integration tests? As far as I know these are supposed to be nearly mandatory for new modules anyways: https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_checklist.html

Reply:
I see what you mean !
Don't worry i didn't install ansible on my managed machine ;) 
I launch the tasks on my localhost because my kubernetes cluster is accessible from it
But i forgot this use case !
I'm agree with you that the yaml import must be in the requirement list !
Thanks 

I will checks for tests also !

Reply:
Integration tests have been added